generator/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3-alpine
+
+COPY requirements.txt /tmp/requirements.txt
+
+RUN pip install -r /tmp/requirements.txt
+
+COPY generate_events.py /var/app/generator.py
```

Comment:
Tiny thing, but why rename `generate_events.py` to `generator.py`?

Reply:
Changed. Now they have the same name.